src/pages/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,9 @@
+
+
+const HomePage = () => {
+    return (
```

Comment:
very silly nit: either put a space between these two lines like the other pages or take the space out of the other pages